frontend/packages/core/src/Input/select.tsx

```diff
@@ -62,7 +56,7 @@ const SelectIcon = (props: any) => (
 
 const BaseSelect = ({ className, ...props }: MuiSelectProps) => (
   <MuiSelect
-    disableUnderline
+    // disableUnderline
```

Comment:
just curious - what are the tradeoffs of commenting this out vs just deleting it?

Reply:
Probably just forgot to delete it honestly :) Let me remove it